### PR TITLE
Don't use Rich for confirmation prompt

### PR DESCRIPTION
**Pull request by jcardonnet**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: packages/syft/src/syft/util/util.py [hunk 1]
```diff
@@ -451,11 +450,15 @@ def prompt_warning_message(message: str, confirm: bool = False) -> bool:
     warning = SyftWarning(message=message)
     display(warning)
 
-    if confirm:
-        allowed = Confirm.ask("Would you like to proceed?")
-        if not allowed:
+    while confirm:
```
Comment (tcp) [inline packages/syft/src/syft/util/util.py:453]: Won't this lock the loop? @jcardonnet

**Review by tcp — COMMENTED** [review]
@jcardonnet left a question for ya :)

**Comment by jcardonnet** [comment]
@tcp If confirm is True it will keep asking until you reply Y or N and then exit the loop. If it's False, it will skip the loop and return immediately

File: packages/syft/src/syft/util/util.py (hunk 1 above)
Comment (jcardonnet) [reply packages/syft/src/syft/util/util.py:453]: @tcp I reproduced the way Rich behaves. It will keep asking until you reply Y or N.

**Review by tcp — APPROVED** [review]

**Pull request merged**